l10n_ch_qr_bill_scan/tests/test_scan_qrbill.py

```diff
@@ -0,0 +1,257 @@
+    "CH",
+    "",
+    "",
+    "",
+    "",
+    "",
+    "",
+    "",
+    "1949.75",
+    "CHF",
+    "S",
+    "Your company",
+    "Bahnhofplatz",
+    "10",
+    "3011",
+    "Bern",
+    "CH",
+    "{ref_type}",
+    "{ref}",
+    "Instruction of 15.09.2019",
+    "EPD",
+    "",
+    "",
+    "",
+]
+
+
+class TestScanQRBill(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env.user.company_id.invoice_import_create_bank_account = True
+        cls.env.ref("base.CHF").active = True
+        cls.supplier = cls.env["res.partner"].create(
+            {
+                "name": "Camptocamp",
+                "street": "EPFL Innovation Park",
+                "street2": "Bldg A",
+                "zip": "1015",
+                "city": "Lausanne",
+                "country_id": cls.env.ref("base.ch").id,
+            }
+        )
+        cls.supplier.supplier_rank = 1
+
+        cls.expense_account = cls.env["account.account"].create(
+            {
+                "code": "612AII",
+                "name": "expense account invoice import",
+                "user_type_id": cls.env.ref("account.data_account_type_expenses").id,
+            }
+        )
+        cls.env["account.invoice.import.config"].create(
+            {
+                "name": "Camptocamp - one line no product",
+                "partner_id": cls.supplier.id,
+                "invoice_line_method": "1line_no_product",
+                "account_id": cls.expense_account.id,
+            }
+        )
+
+    def wiz_import_invoice_file(self, file_path, file_name):
+        """ Import a file of a vendor bill """
+        with tools.file_open(file_path, "rb") as f:
+            invoice_file = base64.b64encode(f.read())
+        wiz = self.env["account.invoice.import"].create({})
+        wiz.invoice_file = invoice_file
+        wiz.invoice_filename = file_name
+        return wiz
+
+    def import_invoice_file(self, file_path, file_name):
+        """Import scanned data from a vendor bill
+
+        And return the created invoice
+        """
+        wiz = self.wiz_import_invoice_file(file_path, file_name)
+        res = wiz.import_invoice()
+        if res.get("res_model") == "account.move":
+            invoice = self.env["account.move"].browse(res["res_id"])
+            return invoice
+        return None
+
+    def wiz_import_invoice_scan(self, invoice_scan):
+        """ Import scanned data from a vendor bill """
+        wiz = self.env["account.invoice.import"].create({})
+        wiz.invoice_scan = invoice_scan
+        return wiz
+
+    def import_invoice_scan(self, invoice_scan):
+        """Import scanned data from a vendor bill
+
+        And return the created invoice
+        """
+        wiz = self.wiz_import_invoice_scan(invoice_scan)
+        res = wiz.import_invoice()
+        if res.get("res_model") == "account.move":
+            invoice = self.env["account.move"].browse(res["res_id"])
```

Comment:
wrong model :fire:

Reply:
Maybe you know how to solve Travis build issue?
I tried to add python requirements to travis.yml. Didn't help.